client/app/scripts/actions/app-actions.js

```diff
@@ -635,13 +635,15 @@ function updateFromNodesDeltaBuffer(dispatch, state) {
 }
 
 export function clickResumeUpdate() {
-  return (dispatch, getServiceState) => {
+  return (dispatch, getState) => {
     dispatch({
       type: ActionTypes.CLICK_RESUME_UPDATE
     });
+    // TODO: Find a better way to do this (see the comment above).
+    const state = getState().scope || getState();
```

Comment:
I'd say the whole point of this PR was to move the pause button back to the footer so that it's not a part of `<CloudFeature />`, so that it would stay visible in Scope standalone as was the case so far. :)